scripts/release/build.js

```diff
@@ -26,9 +26,9 @@ async function writeData() {
   await fs.writeFile(dest, data);
 }
 
-async function writeIndex() {
-  const dest = path.resolve(directory, 'index.js');
-  const content = `module.exports = require("./data.json");\n`;
+async function writeWrapper() {
+  const dest = path.resolve(directory, 'nodewrapper.mjs');
+  const content = `import fs from 'node:fs';\nconst bcd = JSON.parse(fs.readFileSync(new URL('./data.json', import.meta.url)));\nexport default bcd;\n`;
```

Comment:
Maybe for readability the newlines could be as is:

```suggestion
  const content = `import fs from 'node:fs';
const bcd = JSON.parse(fs.readFileSync(new URL('./data.json', import.meta.url)));
export default bcd;
`;
```